expressEngine/product/service/product.js

```diff
@@ -27,6 +27,29 @@ class Service {
     return { data: product, message: success.productRegistered };
   }
 
+  updateProduct = async (data) => {
+    const { criteria, dataToUpdate } = data;
+    if (!criteria || Object.entries(criteria).length === 0) {
+      return { error: error.emptyCriteria };
+    }
+    if (!dataToUpdate || Object.entries(dataToUpdate).length === 0) {
+      return { error: error.emptyData };
+    }
+    const collection = Product;
+    const product = await DBService.updateOne({ collection, dataToUpdate, criteria });
+    if (product.error) {
+      return product;
+    }
+    const { nModified: FilesModifiled, n: FilesFound } = product;
```

Comment:
@aman-successive use camelCase for variable names

Reply:
updated it